pyEDAA/UCIS/Cobertura.py

```diff
@@ -94,40 +103,47 @@ class Package:
 
 	name: str
 	classes: Dict[str, Class]
-	lines_valid: int
-	lines_covered: int
+	linesValid: int
+	linesCovered: int
 
 	def __init__(self, name: str):
 		self.name = name
 		self.classes = {}
-		self.lines_valid = 0
-		self.lines_covered = 0
+		self.linesValid = 0
+		self.linesCovered = 0
 
-	def add_statement(self, class_name: str, source_file: str, line: int, count: int) -> None:
-		try:
-			self.classes[class_name].add_statement(line, count)
-		except KeyError:
-			self.classes[class_name] = Class(source_file)
-			self.classes.get(class_name).add_statement(line, count)
+	def addClass(self, coberturaClass: Class):
+		assert coberturaClass.name not in self.classes, "Duplicated class name"
```

Comment:
Please convert assertions to proper exceptions.

I suggest to create a `CoberturaException`.

Reply:
Fixed in https://github.com/edaa-org/pyEDAA.UCIS/pull/7/commits/2b8e88b0a0313af3046a39ecf542d45e68d6387a

Reply:
Further modifications requested.